mcs/class/corlib/corlib_test.dll.sources

```diff
@@ -86,6 +86,7 @@ System/EnumTest.cs
 System/EnvironmentTest.cs
 System/ExceptionTest.cs
 System/FloatingPointFormatterTest.cs
+System.GC/GCtest.cs
```

Comment:
This is GCTest. You should try out changes locally : make check

Reply:
I just noticed there is also a System/GCTest. You could simply move the entire test case in that file